### PR TITLE
bandwhich: add page

### PR DESCRIPTION
**Pull request by Waples**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/bandwhich.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# bandwhich
+
+> A CLI utility for displaying current network utilization by process, connection and remote IP/hostname.
```
Comment (bl-ue) [inline pages/common/bandwhich.md:3]:
```suggestion
> A CLI utility for displaying current network utilization by process, connection or remote IP/hostname.
```

File: pages/common/bandwhich.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/imsnif/bandwhich>.
+
+- Show remote addresses table only:
```
Comment (bl-ue) [inline pages/common/bandwhich.md:6]:
```suggestion
- Show the remote addresses table only:
```

File: pages/common/bandwhich.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`bandwhich --addresses`
+
+- Show DNS queries:
+
+`bandwhich --show-dns`
+
+- Show total (cumulative) usages:
```
Comment (bl-ue) [inline pages/common/bandwhich.md:14]:
```suggestion
- Show total (cumulative) usage:
```

File: pages/common/bandwhich.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`bandwhich --total-utilization`
+
+- Show network utilization for given interface (eg. {{eth0}}):
+
+`bandwhich --interface {{interface}}`
```
Comment (bl-ue) [inline pages/common/bandwhich.md:20]:
```suggestion
- Show network utilization for a specific network interface:

`bandwhich --interface {{eth0}}`
```

File: pages/common/bandwhich.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show DNS queries with a given DNS server:
+
+`bandwhich --show-dns --dns-server {{dns-server}}`
```
Comment (bl-ue) [inline pages/common/bandwhich.md:24]:
```suggestion
- Show DNS queries for a specific DNS server:

`bandwhich --show-dns --dns-server {{dns_server_ip}}`
```

Comment (Waples) [reply pages/common/bandwhich.md:24]: @bl-ue It doesn't show DNS quiries _"for_" a specific DNS server, it uses the given DNS server.

Comment (bl-ue) [reply pages/common/bandwhich.md:24]: (sorry, forgot about this one) Updated.  Is it correct now?

Comment (Waples) [reply pages/common/bandwhich.md:24]: I'm a bit 50-50; it *uses* the specified DNS server, your sentence makes it seem like doing queries to the DNS server. It's a bit, "how you read it, then it makes sense"...

Comment (sbrl) [reply pages/common/bandwhich.md:24]: Ah, I see @Waples. What about this?

```suggestion
- Show DNS queries with a given DNS server:

`bandwhich --show-dns --dns-server {{dns_server_ip}}`
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/common/bandwhich.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# bandwhich
+
+> A CLI utility for displaying current network utilization by process, connection or remote IP/hostname.
+> More information: <https://github.com/imsnif/bandwhich>.
+
+- Show the remote addresses table only:
+
+`bandwhich --addresses`
+
+- Show DNS queries:
```
[the remark this answers is not in the document]
Comment (Waples) [reply pages/common/bandwhich.md:10]: I think the latter one; haven't played around with this allot yet.

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Waples!

**Pull request merged**